.travis.yml

```diff
@@ -1,28 +1,32 @@
 language: python
+
 python:
 - '3.6'
 - '3.5'
 - '3.4'
 
 matrix:
   include:
-    - python: 3.6
-      sudo: required
-      dist: trusty
+    - python: '3.4'
       env:
-      - TOXENV=style
+      - DNF=yum OS=centos7
       before_install:
-      - sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
-      - sudo rm -rf /etc/apt/sources.list.d/*
-      - sudo apt-get -qq update && sudo apt-get -qq install shellcheck
+      - sed -i 's@fedora@jsmigel/centos-epel@' Dockerfile
+    - python: '3.6'
+      env:
+      - TOXENV=style EXTRA="ShellCheck which"
 
-cache:
-  directories:
-  - $HOME/.cache/pip
-  - .eggs
+sudo: required
+dist: trusty
+
+services:
+  - docker
 
 install:
-- pip install --upgrade pip setuptools
-- pip install tox-travis
+  - docker build --build-arg PYTHON=python${TRAVIS_PYTHON_VERSION} --build-arg EXTRA="$EXTRA" --build-arg DNF=${DNF:=dnf} -t rpmlb .
```

Comment:
OK.
Another merit to create the script file is that people can try to run the docker test on local environment easily.
But in this PR, I agree with you.